README.md

```diff
@@ -40,6 +40,23 @@ The following command will restart all pods in the `web` and `jobs` deployments:
 
 `kubernetes-restart <kube namespace> <kube context> --deployments=web,jobs`
 
+### Running one off tasks
+
+To trigger a single use job such as a rake task, use the following command:
+
+`kubernetes-run <kube namespace> <kube context> --command=rake <arguments>`
+
+This command assumes that there is a `ConfigMap` named `task-runner-template`, which contains a full pod specification in it's `data`. The pod specification in turn must have a container named `task-runner`. This container will be passed the arguments and command passed to `kubernetes-run`.
```

Comment:
Could we provide a piece of documentation on how to put the `task-runner-template` in `ConfigMap`?